.github/workflows/docker.yml

```diff
@@ -1,13 +1,8 @@
 name: Docker build
-
+# This workflow will only build for Pull Requests which are opened, changed , merged 
 on:
-  push:
```

Comment:
Could you please describe the intention of removing these lines for `push` and `branches-ignore`? Want to make sure I'm understanding correctly. 

Until we resolve our unit tests so that the 'CI' workflow fully completes successfully, I think we will want to maintain the behavior of docker images being built by this 'docker' workflow, so that we can at least check that the carma-platform docker image builds successfully during CI.

Reply:
We will revert all this changes after fixing the Platform build. This changes will only reflect on feature/add_gha branch that we are working.

Reply:
To follow-up on this, can we revert this prior to merging?

Reply:
removed Push trigger as we only build docker.yml on PR's